# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-video-nouveau/nv10-hack.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-nouveau/src/nv10_exa.c.vanilla	2010-01-17 14:32:26.000000000 +0100
+++ xf86-video-nouveau/src/nv10_exa.c	2010-01-17 14:32:43.000000000 +0100
@@ -488,7 +488,7 @@
 	struct nouveau_channel *chan = pNv->chan;
 	struct nouveau_grobj *celsius = pNv->Nv3D;
 	uint32_t rc_in_alpha = 0, rc_in_rgb = 0;
-
+#if 0
 	if (PICT_FORMAT_A(src->format))
 		rc_in_alpha |= RC_IN_TEX(A, ALPHA, 0);
 	else
@@ -534,6 +534,7 @@
 	OUT_RING  (chan, rc_in_alpha);
 	BEGIN_RING(chan, celsius, NV10TCL_RC_IN_RGB(0), 1);
 	OUT_RING  (chan, rc_in_rgb);
+#endif
 }
 
 static void
